src/Symfony/Component/Translation/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.4
+---
+
+ * Add checking for id / source mismatches between base and locale in `translation-status.php` script
```

Comment:
As this is an internal tool, I would remove this. That's also why @nicolas-grekas asks to move it to 4.4 (we are using 4.4 for all new translations).

Reply:
Done!